src/Equinor.ProCoSys.IPO.WebApi/Controllers/Me/MeController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Query.GetOutstandingIpos;
+using Equinor.ProCoSys.IPO.WebApi.Middleware;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using ServiceResult.ApiExtensions;
+
+namespace Equinor.ProCoSys.IPO.WebApi.Controllers.Me
+{
+    [ApiController]
+    [Route("Me")]
+    public class MeController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public MeController(IMediator mediator) => _mediator = mediator;
+
+        [Authorize(Roles = Permissions.IPO_READ)]
+        [HttpGet("OutstandingIpos")]
+        public async Task<ActionResult<OutstandingIposResultDto>> GetOutstandingIpos(
+            [FromHeader(Name = CurrentPlantMiddleware.PlantHeader)]
+            [Required]
+            string plant,
+            [FromQuery] string projectName)
```

Comment:
What do I miss here ..... this new feature is for showing all outstanding ipo todo in main ProCoSys, right?
Isn't that UX-component at ProCoSys frontpage? ... In that context, user haven't chosen a project, so how will You implement the query using OutstandingIpos endpoint?

Reply:
Shoot this is true. Mistake on my end... Do you see any problem with rewriting the endpoint to not take projectName?

Reply:
> Do you see any problem with rewriting the endpoint to not take projectName?
No .. that must be the correct solution

Reply:
Confirmed this with the client, scenario is so unlikely there is no need for sorting on project access. Will therefore show outstanding IPOs for all projects in current plant.

Reply:
Add a comment inside Handler explaining this business decision. Say that we COULD have filtered on project access, but blah blah